app/routes/api/get-all-grouped-by-sbi-numbers.js

```diff
@@ -0,0 +1,35 @@
+const Joi = require('joi')
+const Boom = require('@hapi/boom')
+const { getAllGroupedBySbiNumbers } = require('../../repositories/application-repository')
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/api/application/getAllGroupedBySbiNumbers',
```

Comment:
not 100% sure on the naming (will have a think)

Also could we maybe add a `docs` folder in the route of the repository similar to how we have done for eligibility please with the OpenAPI? :) 

https://github.com/DEFRA/ffc-ahwr-eligibility/tree/main/docs